_posts/elements/2015-02-04-example-kitchen-sink.md

```diff
@@ -17,33 +17,33 @@ info:  Vestibulum id ligula porta felis euismod semper. Donec id elit non mi por
 
```

Comment:
I think this post can be deleted - it was originally created to make sure my PL doc site styling was scoped to not affect PL elements.